packages/xo-server-audit/src/index.js

```diff
@@ -100,9 +101,9 @@ class Db extends Storage {
     await this._db.del(id)
   }
 
-  get(id) {
+  get(id, throwIfNotFound = false) {
```

Comment:
Why this change?

Reply:
To check if the records exists. Please see https://github.com/vatesfr/xen-orchestra/pull/5277/files#diff-9d978bd643fbbf52bc97eead8614a9ba02a45656e1cae39cde1776560d1b8a02R158-R159.

I can use `assert` instead, i just did it because throwing in case of not found is a common behavior of DB's get.